pandas/__init__.py

```diff
@@ -24,7 +24,7 @@
 try:
     from pandas import hashtable, tslib, lib
 except ImportError as e:  # pragma: no cover
-    module = str(e).lstrip('cannot import name ')  # hack but overkill to use re
+    module = str(e).split('cannot import name ', 1)[-1]
     raise ImportError("C extension: {0} not built. If you want to import "
```

Comment:
Because in [this check](https://ci.appveyor.com/project/jreback/pandas-465/build/1.0.1766/job/hscus0h1el92sq0h) `lstrip` stripped `cannot import name total_seconds` into `l_seconds`. (`l` is the first character in the error string not present in `cannot import name `.)